src/serac/infrastructure/tests/serac_error_handling.cpp

```diff
@@ -54,21 +54,17 @@ TEST(SeracErrorHandling, EquationSolverKinsolNotAvailable)
 }
 #endif
 
-TEST(SeracErrorHandling, BcProjectRequiresState)
-{
-  auto mesh      = mesh::refineAndDistribute(buildDiskMesh(10));
-  int  num_attrs = mesh->bdr_attributes.Max();
-
-  auto              coef = std::make_shared<mfem::ConstantCoefficient>();
-  BoundaryCondition bc(coef, 0, std::set<int>{1}, num_attrs);
-  EXPECT_THROW(bc.project(), SlicErrorException);
-}
-
 TEST(SeracErrorHandling, BcOneComponentVectorCoef)
```

Comment:
FWIW, I'm a fan of very descriptive test names, favoring clarity to brevity. I prefer sentence-length names that describe exactly what's being tested, so that when it fails, anyone can understand what went wrong. It's hard to know what this test is actually testing as it stands. You have to read the test logic to know. I'm not requesting you to change this now, just making a remark while it's topical.